crates/core/tedge_agent/src/agent.rs

```diff
@@ -275,13 +298,19 @@ impl SmAgent {
 
         self.process_pending_operation(&mut mqtt.published).await?;
 
+        // spawning file transfer server
+        tokio::spawn(async move {
+            if let Err(err) = server.await {
+                error!("{}", err);
```

Comment:
Curious to know when this logging will happen. If the `get` or `put` functions fail with a `FileTransferError`, for e.g: when the path contains special characters like `..`, would that trigger this log? If yes, that would be a problem as the server may not process anymore requests.